lib/atp_scraper/activity.rb

```diff
@@ -67,5 +67,10 @@ def pickup_player_name
         .css("meta[property=\"pageTransitionTitle\"]")
         .attr("content").value
     end
+
+    def convert_ranking(ranking)
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)